example_project/settings.py

```diff
@@ -76,7 +76,6 @@
 
 PASSWORD_HASHERS = (
     'django.contrib.auth.hashers.MD5PasswordHasher',
```

Comment:
I realize this is mostly irrelevant since this is an example app, and this MR does not add this hasher, but I would suggest we change this to use PBKDF2PasswordHasher which has been available from 3.2 onward.  Django lists this as not recommended as far back as 3.2 [not recommended](https://github.com/django/django/blob/c3d3af8ea369e16263c8330b1745ba101d1c8d62/django/contrib/auth/hashers.py#L703)

Reply:
Changed.